build/rpmbuild-docker/Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM rockylinux:8
 RUN yum -y install epel-release nodejs
 RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
 RUN yum -y install yum-utils rpmdevtools createrepo_c mock git golang yarn
+RUN export GOBIN=$(go env GOPATH)/bin
+RUN mkdir $GOBIN
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
Yes, that's possible, I'll change it this way.

Reply:
Sorry, I made some force pushes, which might cause it lose tracking of changes.
I made it building from local source instead of installing from network, see this code

[code](https://github.com/shatteredsilicon/ssm-submodules/pull/1/files#diff-ec7ecd42bb4c8c87165b2de4aa13525dde4b372380c0b2a42df828673905abcbR48-R52)
```diff
+ ENV GOPATH=/home/builder/go
+ RUN mkdir -p ${GOPATH}/src/github.com/golang
+ ADD dep ${GOPATH}/src/github.com/golang/dep
+ RUN GO111MODULE=off go install -ldflags="-X main.version=v0.5.4" 
+ ${GOPATH}/src/github.com/golang/dep/cmd/dep
+ ENV PATH=$PATH:${GOPATH}/bin
```